iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -18,7 +18,6 @@
 #include "interactive/interactive_query_cli.hpp"
```

Comment:
In another PR, a commit of removing the CLI implementation will be dropped and new CLI implementation will be restored.